server/src/main/java/io/deephaven/server/session/SessionServiceGrpcImpl.java

```diff
@@ -65,10 +65,14 @@ public SessionServiceGrpcImpl(final SessionService service, final TicketRouter t
     @Override
     public void newSession(final HandshakeRequest request, final StreamObserver<HandshakeResponse> responseObserver) {
         GrpcUtil.rpcWrapper(log, responseObserver, () -> {
-            // TODO: once jsapi is updated to use flight auth, then newSession can be deprecated or removed
-            final AuthContext authContext = new AuthContext.SuperUser();
+            // TODO: once client APIs are updated to use flight auth, then newSession can be deprecated or removed
```

Comment:
going to revert changes in this file, the old impl "works" and any change here will break existing clients.